zimui/src/components/ErrorDisplay.vue

```diff
@@ -8,25 +8,81 @@ const errimage = ref(errimageData)
 </script>
 
 <template>
-  <div class="error-container">
-    <img :src="errimage" class="error-image" />
-    <p class="error-text">{{ main.errorMessage }}</p>
-    <p class="error-text">{{ main.errorDetails }}</p>
+  <div class="error-page">
+    <div class="error-container">
+      <div class="error-content">
+        <p class="error-text">{{ main.errorMessage }}</p>
+        <p class="error-text">{{ main.errorDetails }}</p>
+      </div>
+      <div class="error-image-container">
+        <img :src="errimage" class="error-image" alt="error image" />
+      </div>
+    </div>
   </div>
 </template>
 
 <style scoped>
-.error-container {
+body {
```

Comment:
What is this used for? To me, it looks like you cannot modify body style from a scoped style

Reply:
ohh i forget to remove this, i was trying something.

Reply:
done.